aim/sdk/uri_service.py

```diff
@@ -52,7 +53,10 @@ def request_batch(self, uri_batch: List[str]) -> Iterator[Dict[str, bytes]]:
 
             # TODO: [MV] change to some other implementation of view when available
             #  which won't collect in case of custom objects
-            yield {uri: container.tree().view(resource_path).collect()}
+            data = container.tree().subtree(resource_path).collect()
```

Comment:
Why not to use [resource_path] instead?
Also, just by looking at the implementation of `decode_uri()` it looks like `resource_path` is always string :/ Am I correct?

Reply:
For the first part: yeah, need to be changed. 
For the second part: because the URIs need to be transferred to the UI and back in the request body, that's why I sticked to strings.

Reply:
@mahnerak we have a difference in results for the following two expressions:
`container.tree().subtree(resource_path).collect9)` and
`container.tree().collect(resource_path)`

lets say `resourse_path == a.b.c`
the 2nd one will return `{a: {b: {c: <VALUE>}}}`

Reply:
for the above example
`container.tree().subtree('a').collect(('b', 'c'))` will return `<VALUE>`. I assume this comes from difference in implementation of PrefixView and RocksContainer